src/main/scala/codacy/plugins/test/JsonTests.scala

```diff
@@ -12,7 +12,8 @@ object JsonTests extends ITest {
 
   val opt = "json"
 
-  def run(testSources: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
+  def run(testDirectories: Seq[Path], dockerImage: DockerImage, optArgs: Seq[String]): Boolean = {
+    val testSources = testDirectories.filter(_.getFileName.toString == DockerHelpers.testsDirectoryName)
```

Comment:
can you explain the reasoning behind this?
the API accepts a list of directories and then you filter out only `DockerHelpers.testsDirectoryName`?

Reply:
should we remove the argument from the method signature?